### DISCUSSION TITLE
Scrollable Panel

### DISCUSSION THREAD

**Discussion by lserranoruber** [rexrainbow/phaser3-rex-notes#167] [Q&A] [answered]
I'm trying to make a scrollable panel in phaser and I came across [this ](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/ui-scrollablepanel/) page of yours which links to a directory `examples` which no longer exists in this repo. Where can I find the sample code (or something similar) that was in `examples/ui-scollablepanel`? 

Thank you!

**Comment by rexrainbow** [comment, marked as answer]
Thanks for reporting this issue, the [link](https://github.com/rexrainbow/phaser3-rex-notes/tree/master/examples/ui-scrollablepanel) has been fixed.

**Reply by lserranoruber** [reply]
thanks!
